.github/workflows/build-workflow.yml

```diff
@@ -143,7 +143,7 @@ jobs:
             path: target/${{ matrix.target }}/packages/*.*
 
   # Note: Validate that all build jobs completed successfully
-  # If a Runner failues on the setup, it can leave a job marked as "skipped"
+  # If a Runner failures on the setup, it can leave a job marked as "skipped"
```

Comment:
Shouldn't this be the following?

```
If a Runner fails during the setup ...
```

Reply:
thanks, I should of actually read it first.